components/automate-ui/src/app/pages/automate-settings/automate-settings.component.ts

```diff
@@ -178,6 +179,19 @@ export class AutomateSettingsComponent implements OnInit, OnDestroy {
     return allowedKeys.includes(key);
   }
 
+  // Update the width of input when greater than one number
+  public autoUpdateInputWidth(element: HTMLInputElement): void {
+    // Keep default in sync with input width in automate-settings.component.scss
+    const DEFAULT_WIDTH = 64;
+    const length = element.value.length;
+
+    if (length > 1) {
+      element.style.width = `${DEFAULT_WIDTH + (9 * (length - 1))}px`;
```

Comment:
We are guaranteed that characters are 9 pixels wide or is this an approximation?

Reply:
this is an approximation that errs on the side of wider